convert2rhel/systeminfo.py

```diff
@@ -390,7 +390,9 @@ def generate_rpm_va(self, log_filename=PRE_RPM_VA_LOG_FILENAME):
             " minutes. It can be disabled by using the"
             " --no-rpm-va option."
         )
-        rpm_va, _ = utils.run_subprocess(["rpm", "-Va"], print_output=False)
+        # To make convert2rhel faster, `--nodeps` was introduced in the command. This will skip
+        # verification of package dependencies. Discussed in RHELC-1427
```

Comment:
I would rewrite this a bit to something like: 

```suggestion
        # Discussed under RHELC-1427, `--nodeps` will skip verification of package dependencies, avoiding unnecessary packages lookup.
```

I don't think it's necessary to mention that this will speed up convert2rhel.